Manager/UserManager.php

```diff
@@ -192,4 +221,42 @@ public function deleteUser(UserInterface $user)
         $this->entityManager->remove($user);
         $this->entityManager->flush();
     }
+
+    /**
+     * @param User $user
+     * @return mixed
+     * @throws TransportExceptionInterface
+     */
+    public function sendInvitation(User $user)
+    {
+        if (null !== $user) {
+            $fromEmail = $this->parameterBag->get('webstack_user.registration.from_email');
+
+            if (null === $user->getConfirmationToken()) {
+                $user->setConfirmationToken($this->tokenGenerator->generateToken());
+            }
+
+            $user->setPasswordRequestedAt(new DateTime());
+
+            $this->entityManager->flush();
+
+            $email = (new TemplatedEmail())
+                ->from(new Address($fromEmail['address'], $fromEmail['sender_name']))
+                ->to($user->getEmail())
```

Comment:
Hier kunnen we toch ook new Address gebruiken en dan voor en achternaam samenvoegen in de tweede parameter